echopype/convert/parse_ek60.py

```diff
@@ -14,3 +17,32 @@ def __init__(
         **kwargs,
     ):
         super().__init__(file, bot_file, idx_file, storage_options, sonar_model)
+
+
+def is_ER60(raw_file, storage_options):
+    """Check if a raw data file is from Simrad EK60 echosounder."""
+    with RawSimradFile(raw_file, "r", storage_options=storage_options) as fid:
+        config_datagram = fid.read(1)
+        config_datagram["timestamp"] = np.datetime64(
+            config_datagram["timestamp"].replace(tzinfo=None), "[ns]"
+        )
+        # Return True if the sounder name matches "ER60"
+        try:
+            return config_datagram["sounder_name"] in {"ER60", "EK60"}
+        except KeyError:
+            return False
+
+
+def is_EK60(raw_file, storage_options):
+    """Check if a raw data file is from Simrad EK60 echosounder."""
+    with RawSimradFile(raw_file, "r", storage_options=storage_options) as fid:
+        config_datagram = fid.read(1)
+        config_datagram["timestamp"] = np.datetime64(
+            config_datagram["timestamp"].replace(tzinfo=None), "[ns]"
+        )
```

Comment:
I am not sure why you are accessing the timestamp here. 

See Gavin's https://github.com/OSOceanAcoustics/echopype/issues/494#issuecomment-971917322 that I think would be useful here to check the format without using the full blown parser. Implementing that into the checker would address #494 too in this PR.